### DISCUSSION TITLE
External library settings

### DISCUSSION THREAD

**Discussion by snatch84** [immich-app/immich#9503] [Q&A] [answered]
Good morning everyone, I have a question, but if I set a external path with :ro, in theory, shouldn't it disable the delete button on the app / web page?
I'm on a Windows and I set volumes to "D:/test:mnt/test:ro", i can import external file in that mount, but if i open a photo i can delete that or modify metadata.
if I delete a photo, in reality only the "link" is deleted, because I no longer see it on the app/web, but the file remains in place in folder. If I rescan the library the file reappears.
Is this behavior intentional? 
Shouldn't the delete button be disabled for read-only paths?
thanks

**Comment by bo0tzz** [comment, marked as answer]
We considered this, but it would be fairly complex to get right so we decided not to.
